src/facebook/facebookdatatypesyncadaptor.h

```diff
@@ -71,6 +72,10 @@ private Q_SLOTS:
     void signIn(Accounts::Account *account);
     bool m_triedLoading; // Is true if we tried to load (even if we failed)
     QString m_clientId;
+    QString m_graphAPI;
 };
 
+inline QString FacebookDataTypeSyncAdaptor::graphAPI() const
+{ return m_graphAPI; }
```

Comment:
I'd prefer if the implementation was put into the .cpp unless we're 100% certain that the implementation of this function won't change in the future.